### PR TITLE
gemma 0.98

### PR DESCRIPTION
**Pull request by pjotrp**
* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by pjotrp** [comment]
Why is Circleci not running the macos tests?

**Comment by chapmanb** [comment]
I could cancel and restart the MacOS job but it looks to me from recent Bioconda PRs that they're all waiting on OSX so I'm worried doing so would put you back at the end of the queue. I'm not sure what is wrong with the long runtimes but if it were my job I'd just wait and keep an eye on it.

**Comment by pjotrp** [comment]
OK, I rebased and tests should run again.

File: recipes/gemma/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    - {{ compiler('cxx') }}
+  host:
+    - openblas
+    - eigen
+    - gsl
+    - zlib
+  run:
+    - openblas
+    - gsl
+    - zlib
```
Comment (bgruening) [inline recipes/gemma/meta.yaml]: Same for zlib, it might not be needed anymore as runtime dependency.

**Comment by pjotrp** [comment]
OK, I'll try again.

**Comment by pjotrp** [comment]
lgtm

**Pull request merged**